Editorial: Enhance Windows Installation Instructions

snuggs opened the pull request:
Addresses:
    - https://github.com/w3c/web-platform-tests/issues/6876#issuecomment-334187748

  - [x] Fix misspellings in WIndows installation
  - [x] Better Windows examples as per @domenic's request

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/w3c/web-platform-tests/8634)
<!-- Reviewable:end -->

snuggs committed:
Editorial: Enhance Windows Installation Instructions

  Addresses:
    - https://github.com/w3c/web-platform-tests/issues/6876#issuecomment-334187748

  - [x] Fix misspellings in WIndows installation
  - [x] Better Windows examples as per @domenic's request

w3c-bots commented:
# Build PASSED

Started: 2017-12-08 14:31:31
Finished: 2017-12-08 14:37:32


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21671)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/313512125)

domenic commented on README.md:
````diff
@@ -53,8 +53,15 @@ Running Tests Manually
+**On Windows**: You will need to preceed the prior command with
+`python` or the path to the python binary.
+```bash
+c:\python27\python.exe wpt serve
````
Just do `python` here and elsewhere.

snuggs committed:
Update README.md to strip absolute python path

  Addresses:
    - https://github.com/w3c/web-platform-tests/pull/8634#discussion_r155803882

snuggs commented:
@domenic Fixed in https://github.com/w3c/web-platform-tests/pull/8634/commits/44f23002a745631a455bc6e6a441cdfc8ee9697c

domenic reviewed (approved):
LGTM. Will give it a day or so before merging in case @jgraham has any thoughts; I believe he wrote this originally.
